src/Psy/Command/ParseCommand.php

```diff
@@ -59,12 +68,23 @@ public function setPresenter(Presenter $presenter)
      */
     protected function configure()
     {
+        $definition = array(
+            new InputArgument('code', InputArgument::REQUIRED, 'PHP code to parse.'),
+            new InputOption('depth', '', InputOption::VALUE_REQUIRED, 'Depth to parse', 10),
+        );
+        if ($this->parserFactory->hasKindsSupport()) {
+            $definition[] = new InputOption(
+                'parser-kind',
```

Comment:
I'm not sure I love `parser-kind` as an option name. I don't have a better suggestion right now, other than to drop the `parser-` prefix, since that's kind of implied by the command :)